tests/helpers/pretender.js

```diff
@@ -0,0 +1,13 @@
+import Pretender from "pretender";
+
+export const setupPretenderHook = function (hooks) {
```

Comment:
bit of a nitpick but this way it's more in line with other similar setup functions:

```suggestion
export const setupPretender = function (hooks) {
```